shell_integration/dolphin/CMakeLists.txt

```diff
@@ -2,10 +2,9 @@ project(dolphin-owncloud)
 
 cmake_minimum_required(VERSION 2.8.12)
 include(FeatureSummary)
-set(QT_MIN_VERSION "5.3.0")
 set(KF5_MIN_VERSION "5.16.0")
 
-find_package(Qt5 ${QT_MIN_VERSION} CONFIG REQUIRED COMPONENTS Core Network)
```

Comment:
We need the system Qt here which could be anything.

@dschmidt why do we build the dolphin stuff as part of the main project?

Reply:
We don't need to look for some system Qt when building this sub-project with the remaining project IMO. This file declares a project anyway, so this can be built separately. What should be preserved is the old minimum Qt version for use in standalone builds.